plugins/com.google.cloud.tools.eclipse.dataflow.ui/src/com/google/cloud/tools/eclipse/dataflow/ui/launcher/DefaultedPipelineOptionsComponent.java

```diff
@@ -236,4 +236,8 @@ public void widgetSelected(SelectionEvent e) {
   public void validate() {
     defaultOptions.validate();
   }
+
+  public void setAccountRequired(boolean required) {
```

Comment:
I don't think this method needs to be public. Yes, there's probably a lot of other currently public methods that don't need to be either. One of these days, I'd really like to clean up the dataflow package.